packages/gestalt-datepicker/src/DatePicker.js

```diff
@@ -11,27 +17,84 @@ import dateFormat from './dateFormat.js';
 import { type LocaleData } from './LocaleDataTypes.js';
 
 type Props = {|
+  /**
+   *
+   */
   disabled?: boolean,
+  /**
+   * See the [error message example](https://gestalt.pinterest.systems/datepicker#errorMessage) to learn more.
+   */
   errorMessage?: string,
+  /**
+   * Array of disabled dates. See the [disabled dates example](https://gestalt.pinterest.systems/datepicker#exclude) to learn more.
+   */
   excludeDates?: $ReadOnlyArray<Date>,
+  /**
+   * More information about how to complete the date picker field. See the [helper text example](https://gestalt.pinterest.systems/datepicker#helperText) to learn more.
+   */
   helperText?: string,
+  /**
+   *
+   */
   id: string,
+  /**
+   * Preferred direction for the calendar popover to open. See the [ideal direction example](https://gestalt.pinterest.systems/datepicker#idealDirection) to learn more.
+   */
   idealDirection?: 'up' | 'right' | 'down' | 'left',
+  /**
+   * Array of enabled dates. See the [enabled dates example](https://gestalt.pinterest.systems/datepicker#include) to learn more.
+   */
   includeDates?: $ReadOnlyArray<Date>,
+  /**
+   *
+   */
   label?: string,
+  /**
+   * DatePicker accepts imported locales from the open source date utility library date-fns. See the [locales example](https://gestalt.pinterest.systems/datepicker#localeData) to learn more.
+   */
   localeData?: LocaleData,
+  /**
+   * Disable dates outside a max date. See the [delimited selection period example](https://gestalt.pinterest.systems/datepicker#maxMinDates) to learn more.
+   */
   maxDate?: Date,
+  /**
+   * Disable dates outside a min date. See the [delimited selection period example](https://gestalt.pinterest.systems/datepicker#maxMinDates) to learn more.
+   */
   minDate?: Date,
-  // $FlowFixMe[unclear-type]
-  nextRef?: ElementRef<*>,
+  /**
+   * Required for date range selection. Pass the complimentary range date picker ref object to DatePicker to autofocus on the unselected date range field. See the [date range picker example](https://gestalt.pinterest.systems/datepicker#rangePicker) to learn more.
+   */
+  nextRef?: Ref<'input'>,
+  /**
+   *
+   */
   onChange: ({|
     event: SyntheticInputEvent<HTMLInputElement>,
     value: Date,
   |}) => void,
+  /**
+   * Date format for locale.
```

Comment:
This doesn't seem correct? This seems like a common use for the placeholder text, but is the text itself not more generic?

Reply:
FIXED.